geode-core/src/jmh/java/org/apache/geode/internal/util/ComputeIfAbsentBenchmark.java

```diff
@@ -64,16 +62,14 @@ public void trialSetup() throws InterruptedException {
     final int numberOfThreads =
         THREAD_POOL_PROCESSOR_MULTIPLE * Runtime.getRuntime().availableProcessors();
 
-    loadGenerationExecutorService =
-        (ScheduledThreadPoolExecutor) Executors.newScheduledThreadPool(
-            numberOfThreads);
-
-    System.out.println(String.format("Pool has %d threads", numberOfThreads));
 
-    loadGenerationExecutorService.setRemoveOnCancelPolicy(true);
+    jmhConcurrencyLoadGenerator = new JmhConcurrencyLoadGenerator(numberOfThreads);
```

Comment:
Curious, why not use the JMH asymmetric thread group feature?

Reply:
Modified test to use the asymmetric thread groups.  Thanks for the suggestion/info!

Reply:
Awesome!